ingest/lambdas/index-rectifier.js

```diff
@@ -0,0 +1,28 @@
+// @ts-check
+const ir = require('../lib/index-rectifier');
+
+const handler = async () => {
+  const region = process.env.AWS_REGION;
+
+  // Perform an Index and DSpace diff.
+  const result = await ir.diff(
+    process.env.OPEN_SEARCH_ENDPOINT,
+    process.env.DSPACE_ENDPOINT,
+    { region }
+  );
+
+  // Queue updated items for re-ingest.
+  await ir.commitUpdatedItems(
+    result.updated,
+    process.env.INGEST_TOPIC_ARN,
+    { region }
+  );
+
+  // Remove deleted items from the index.
+  await ir.commitRemovedItems(
+    result.removed,
+    process.env.OPEN_SEARCH_ENDPOINT
+  );
+};
+
+module.exports = handler;
```

Comment:
Lambda is going to invoke this as bulkIngester.handler, so we need to make sure to export an object containing a handler function rather than exporting a function directly.